packages/venia-ui/lib/components/CartPage/PriceAdjustments/ShippingMethods/__tests__/__snapshots__/shippingRadios.spec.js.snap

```diff
@@ -2,6 +2,7 @@
 
 exports[`automatically selects first option when none selected 1`] = `
 <form
+  onKeyDown={[Function]}
```

Comment:
I got this change on all my `Form` snapshots recently too ... I think this is related to the update to `informed`?

Reply:
Correct.